spec/worker/gamedev_worker_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+describe GamedevWorker do
+  it "enqueues a Gamedev job" do
+    Sidekiq::Testing.fake!
+    expect {
+      GamedevWorker.perform_async
+    }.to change(GamedevWorker.jobs, :size).by(1)
+  end
+
+  context "with succesful response" do
+    it 'creates external posts' do
+      VCR.use_cassette("questions") do
+        Sidekiq::Testing.inline!
+          expect {
+            GamedevWorker.perform_async
+          }.to change(ExternalPost, :count).by(100)
```

Comment:
we have a test for avoiding duplicated external posts
https://github.com/PowerCodeGirls/RubyGameDev.com/blob/master/spec/models/external_post_spec.rb#L4
Do you think it is necessary to add a test for the worker as well?